mods/seasonal/moves.js

```diff
@@ -625,6 +674,28 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Fire",
 	},
+	// explodingdaisies
+	doom: {
+		basePower: 100,
+		accuracy: 100,
+		category: "Special",
+		desc: "",
+		shortDesc: "",
+		id: "doom",
+		name: "DOOM!",
+		pp: 5,
+		priority: 0,
+		flags: {mirror: 1, protect: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, 'Eruption', target);
+			this.add('-anim', source, 'Sunny Day', source);
+		},
+		weather: 'desolateland',
```

Comment:
How does desolate land work when set like this? Is it 5 turns? till switch-out, something else?

Reply:
non primal weather can override it?

Reply:
Change this to sunny day, talked with QC and the set owner and since its basically infinite sunny day lets make it regular sunny day.